src/platforms/android/common/gradle.mdx

```diff
@@ -94,13 +94,31 @@ sentry {
 
       // Specifies a set of instrumentation features that are eligible for bytecode manipulation.
       // Defaults to all available values of InstrumentationFeature enum class.
-      features = [InstrumentationFeature.DATABASE, InstrumentationFeature.FILE_IO]
+      features = [InstrumentationFeature.DATABASE, InstrumentationFeature.FILE_IO, InstrumentationFeature.OKHTTP]
+    }
+    
+    // Enable auto-installation of Sentry components (sentry-android SDK and okhttp, timber and fragment integrations).
+    // Default is enabled.
+    // Only available v3.1.0 and above.
+    autoInstallation {
+      enabled = true
```

Comment:
Above we say default is enabled. Should this config show how to disable instead?

Reply:
mm, I just followed the pattern that we have already here (showing the default values for each option). Otherwise, we'd need to change all of them to align, so I'd rather not

Reply:
Raise an issue so we can follow up later: https://github.com/getsentry/sentry-docs/issues/4845

Reply:
thanks. I was just assuming that people mostly copy without reading the details, so at least this would've been enabled by default.

one concern I have about that approach is how to deal with non-default values for a set (`tracingInstrumentation.features` for instance)? With booleans it's clear, we just flip the flag, but with the set we probably want to show all available options?

Reply:
Probably something like this would do for the set:

```groovy
// Defaults to all available values of InstrumentationFeature enum class.
features = [
  InstrumentationFeature.DATABASE, // only enable database auto-instrumentation
//  InstrumentationFeature.FILE_IO,
//  InstrumentationFeature.OKHTTP,
]
```